mailpoet/lib/Automation/Engine/Control/SubjectTransformerHandler.php

```diff
@@ -97,7 +97,42 @@ function (SubjectTransformer $transformer) use ($key): bool {
   }
 
   /**
```

Comment:
I guess from here down it is for runtime data, and maybe it would be nice to extract it to another class, so the distinction is clear. The class would need to reuse the current class to call subjectKeysForAutomation, I guess. What do you think?

It kind of seems to me that what's above this comment doesn't really belong to the Control namespace, as it's static and used also for validation, but I'm not sure where else to put it 😄

Reply:
Hmm... Yeah, not really sure. It "transforms" subjects, is it maybe a Builder or Mapper? Not sure if I would want to split the "static" and "dynamic" parts. I see where you are coming from :thinking: At least, I would like them to be in the same namespace. It is quite related and I hate it when I have to search for hours for "where was the other part of that again, was it now a validator or a helper or a mapper...."

Reply:
Right, let's keep it in one now. It's a bit of a special case. We can always split it, if we figure we need it.